js/apps/account-ui/maven-resources/theme/keycloak.v3/account/messages/messages_en.properties

```diff
@@ -128,6 +128,8 @@ signedInDevices=Signed in devices
 delete=Delete
 missingPasswordMessage='{{0}}'\: Please specify password.
 otp-help-text=Enter a verification code from authenticator application.
+trusted-device-display-name=Trusted device
+trusted-device-help-text=Trusted devices don't require second factor authentication for a longer period of time
```

Comment:
Maybe make the trusted device period length available to this message as a parameter? (If it isn't already? I'm not so familiar with Keycloak's code)

E.g. so you could display:

> Trusted devices will only require second-factor authentication every 7 days